From 85b6a313960840798dda00b5a15e1badf9d32bcf Mon Sep 17 00:00:00 2001
From: Kent Russell <kent.russell@amd.com>
Date: Tue, 5 Jul 2016 11:08:53 -0400
Subject: [PATCH 1748/2094] drm/amdkfd/kfd_device_queue_manager.c: Remove
 unused lock/out references

The lock is no longer used, and the goto out has also been removed

Change-Id: Iee06a5e0ad8970fdd2e51ad2bbab01ed22d9edea
---
 drivers/gpu/drm/amd/amdkfd/kfd_device_queue_manager.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_device_queue_manager.c b/drivers/gpu/drm/amd/amdkfd/kfd_device_queue_manager.c
index 78033c1..463afb1 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_device_queue_manager.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_device_queue_manager.c
@@ -1110,11 +1110,7 @@ static int unmap_queues_cpsch(struct device_queue_manager *dqm,
 
 	retval = 0;
 
-	if (lock)
-		mutex_lock(&dqm->lock);
 	if (!dqm->active_runlist)
-		goto out;
-	if (dqm->active_runlist == false)
 		return retval;
 
 	pr_debug("kfd: Before destroying queues, sdma queue count is : %u\n",
-- 
2.7.4

